lib/server.js

```diff
@@ -178,4 +280,4 @@ function close () {
   this.close()
 }
 
-module.exports = { createServer }
+module.exports = createServer
```

Comment:
I suggest moving this to line 3. It shows immediately what is being exported from this file. Whereas now, one has to scroll all the way to the bottom to get that information.